File: MODUL-3/C/PRAK304-2310817220002-RAUDATULSHOLEHAH.c
#include <stdio.h>

int main()
{
    int A;
    printf("Input : ");
    scanf("%d", &A);
    {
        if (!A)
        {
            printf("Output : Nol");
        }
        if (A > 0 && A < 10)
        {
            printf("Output : Satuan");
        }
        else if (A >= 10 && A <= 20)
        {
            printf("Output : Belasan");
        }
        else if (A >= 20 && A <= 99)
        {
            printf("Output : Puluhan");
        }
        else if (A >= 100)
        {
            printf("Output : Anda Menginput Melebihi Limit Bilangan");
        }
    }
    return 0;
}

File: MODUL-3/PYTHON/PRAK304-2310817220002-RAUDATULSHOLEHAH.py
A = int(input("input : "))

if A==0:
    print(f"Output: Nol")
elif 0 < A < 10:
    print(f"Output: Satuan")
elif 10 <= A <= 20:
    print(f"Output: Belasan")
elif 20 <= A <=99:
    print(f"Output: Puluhan")
else:
    print(f"Output: Anda Menginput Melebihi Limit Bilangan")